pkg/consts/spd.go

```diff
@@ -42,3 +42,9 @@ const (
 	SPDBaselinePercentMax = 100
 	SPDBaselinePercentMin = 0
 )
+
+// resource names for aggregate metric
```

Comment:
why resource name is defined in `spd.go`?

Reply:
This is just the metric name not the resource name, it has been modified to the metric name.